src/state/lists/updater.ts

```diff
@@ -64,14 +65,8 @@ export default function Updater(): null {
             throw new Error('unexpected no version bump')
           case VersionUpgrade.PATCH:
           case VersionUpgrade.MINOR: {
-            const min = minVersionBump(list.current.tokens, list.pendingUpdate.tokens)
-            // automatically update minor/patch as long as bump matches the min update
-            if (bump >= min) {
+            if (shouldAcceptMinorVersionUpdate(listUrl, list.current, list.pendingUpdate, bump)) {
```

Comment:
nit: this is called for both `patch` and `minor`, consider renaming the function to be more explicit about that.